README.md

```diff
@@ -4,25 +4,33 @@ Welcome to the new Word Javascript API! We hope you enjoy it an find it useful.
 
 ## Release Notes 
 
-This is the first release of JavaScript APIs for Word and we focused on the following functional areas:
- 1. **Basic document navigation:** On top of having access to the user's current selection, we are also providing ways to traverse the document by exposing collection of two of the most important objects in Word: Paragraphs and Content Controls, having easy access to the content of the entire document.
+For this release of the JavaScript APIs for Word we focused on the following functional areas:
+ 1. **Basic document navigation:** On top of having access to the user's current selection, we are also providing ways to traverse the entire document by exposing collections of two of the most important objects in Word: Paragraphs and Content Controls.
 
- 2. **Insertion of content:** Once positioned in a location of the document to add content, we are enabling developers to insert fully formatted content into Word document and capabilities to do append/prepend before/after type of insertions against our main set of objects. Developers can insert either formatted text, HTML, Office Open XML. Developers are also enabled to reuse content from other Word documents by inserting a Word file into the current document.
+ 2. **Insertion of content:** We are enabling developers to insert fully-formatted content into the Word document and to append or prepend inserted content. Developers can insert either formatted text, HTML, or Office Open XML. Developers can also reuse content from other Word documents by inserting a Word file into the current document.
```

Comment:
**Insertion of content:** You can insert fully-formatted HTML, Office Open XML, or text into a Word document. Reuse content from other Word documents by inserting a Word file into the current document.

Insertion implies prepend and append.
Is inserting a Word file the same as work flow as inserting fully-formatted content?